.claude/README.md

```
@@ -0,0 +1,128 @@
+git clone --depth 1 https://github.com/ByronWilliamsCPA/.claude.git /tmp/.claude-update
+cp -r /tmp/.claude-update/agents/*.md .claude/agents/
+cp -r /tmp/.claude-update/commands/*.md .claude/commands/
+cp -r /tmp/.claude-update/skills/* .claude/skills/
+cp -r /tmp/.claude-update/context/*.md .claude/context/
+cp -r /tmp/.claude-update/standards/*.md .claude/standards/
+rm -rf /tmp/.claude-update
+```
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)